CondTools/RunInfo/src/LHCInfoPopConSourceHandler.cc

```diff
@@ -594,27 +362,39 @@ bool LHCInfoPopConSourceHandler::getCTTPSData(cond::persistency::Session& sessio
         if (!lhcCommentAttribute.isNull()) {
           lhcComment = lhcCommentAttribute.data<std::string>();
         }
-        coral::Attribute const& ctppsStatusAttribute = CTPPSDataCursor.currentRow()[std::string("CTPPS_STATUS")];
-        if (!ctppsStatusAttribute.isNull()) {
-          ctppsStatus = ctppsStatusAttribute.data<std::string>();
-        }
+        //coral::Attribute const& ctppsStatusAttribute = CTPPSDataCursor.currentRow()[std::string("CTPPS_STATUS")];
+        //if (!ctppsStatusAttribute.isNull()) {
+        //  ctppsStatus = ctppsStatusAttribute.data<std::string>();
+        //}
```

Comment:
should this be removed?

Reply:
@tvami In principle this has been left intentionally, because the  concerned variables are no longer available in the source db... Are they critical? Should we ask a replacement? Not sure.  But maybe we can remove that block for the moment.

Reply:
Maybe keep them commented but add a sentence above to explain this?